WireUI/Sources/WireMoveToFolderUI/Models/Folder.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+/// Represents a folder in the system
+public struct Folder: Equatable {
+    /// Unique identifier for the folder. Can be nil for temporary folders
+    public let identifier: UUID?
+
+    /// Display name of the folder. Can be nil if not set
+    public let name: String?
```

Comment:
Please avoid redundancy unless there is a significant difference between an empty string and `nil`.
```suggestion
    public let name: String
```

Reply:
Done!